theatre/studio/src/panels/DetailPanel/propEditors/utils/useEditingToolsForPrimitivePropInDetailsPanel.tsx

```diff
@@ -17,49 +17,56 @@ import type {PropTypeConfig} from '@theatre/core/propTypes'
 import {isPropConfSequencable} from '@theatre/shared/propTypes/utils'
 import type {SequenceTrackId} from '@theatre/shared/utils/ids'
 
-interface CommonStuff<T> {
+interface EditingToolsCommon<T> {
   value: T
   beingScrubbed: boolean
   contextMenuItems: Array<IContextMenuItem>
+  /** e.g. `< • >` or `<   >` for {@link Sequenced} */
   controlIndicators: React.ReactElement
 
   temporarilySetValue(v: T): void
   discardTemporaryValue(): void
   permanentlySetValue(v: T): void
 }
 
-interface Default<T> extends CommonStuff<T> {
+interface Default<T> extends EditingToolsCommon<T> {
   type: 'Default'
   shade: Shade
 }
 
-interface Static<T> extends CommonStuff<T> {
+interface Static<T> extends EditingToolsCommon<T> {
   type: 'Static'
   shade: Shade
 }
 
-interface Sequenced<T> extends CommonStuff<T> {
+interface Sequenced<T> extends EditingToolsCommon<T> {
   type: 'Sequenced'
   shade: Shade
+  /** based on the position of the playhead */
   nearbyKeyframes: NearbyKeyframes
 }
 
-type Stuff<T> = Default<T> | Static<T> | Sequenced<T>
+type EditingTools<T> = Default<T> | Static<T> | Sequenced<T>
 
-export function useEditingToolsForPrimitiveProp<
+/**
+ * Notably, this uses the {@link Scrub} api in order to support
+ * indicating which pointers are being scrubbed so indicators of scrubbing
+ * can be presented to everyone.
```

Comment:
What do you mean by "presented" and "everyone"? All parts of studio that access the data?